week-3/F-array-forEach/exercise.js

```diff
@@ -9,6 +9,13 @@
 
 var arr = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15];
 
+arr.forEach(number=>{
+  /********** using ternary operators to return either an empty string or 
+  Fizz / Buzz  concatinating result and addind OR if cancatination returns empty string*/
+  console.log((number%3?"":"Fizz")+(number%5?"":"Buzz")||number)
```

Comment:
This isn't quite right. Have a careful look at the expected output below, especially for the values 10 and 15 in the `arr` array. Are they the same in your output? How could we fix that?

Generally, in this situation I would usually use an `if` statement instead of a ternary. It makes it a little easier to read.

Reply:
i am getting output of Buzz for 10 and FixxBuzz for 15 
which i think is correct in given scenario 10 is not a multiple of 3